packages/core/test/types.test.ts

```diff
@@ -282,19 +258,13 @@ describe('output', () => {
         };
       },
       context: { password: 'okoń' },
```

Comment:
What kind of password is this?

Reply:
😂 I forgot that i sneaked in this polish easter egg here in the past